django/utils/formats.py

```diff
@@ -23,7 +23,11 @@
         '%Y-%m-%d %H:%M:%S',
         '%Y-%m-%d %H:%M:%S.%f',
         '%Y-%m-%d %H:%M',
-        '%Y-%m-%d'
+        '%Y-%m-%d',
+        # ISO 8601
+        '%Y-%m-%dT%H:%M:%S',
+        '%Y-%m-%dT%H:%M:%S.%f',
+        '%Y-%m-%dT%H:%M',
```

Comment:
Can we add support for the [offset and `Z` formats](https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC) too? (They're used **a lot**.)